test/boost/cdc_test.cc

```diff
@@ -23,6 +23,7 @@
 #include <string>
 
 #include "cdc/log.hh"
+#include "cdc/cdc_extension.hh"
```

Comment:
Please merge this patch into the one that breaks the tests and introduces the need for the fixes included in this patch.

Reply:
What about this (and potentially other bisectability issues)?

Reply:
Those issues now should be resolved. For every commit in this patch, I checked that it compiles and makes unit tests pass. Before uploading the most recent version, I did the same.